lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -316,6 +318,21 @@ def _prepare_environment(  # noqa: C901
 
             # check locations
             for req in nodes_requirement:
+                # If no disk type feature is specified, use DiskPremiumLRS
```

Comment:
We need to do it in `_prepare_environment` so that we can return correct environment capability. If we skip this step, we will get environments that can potentially have standard , premium and ephemeral features. However, this isn't possible as VM's can only have one disk at a time.

Reply:
To summarize : If the environment is using default disk type, then it should have no disk feature added to it's capability.

I think this case can be handled in `azure_generate_min_capability`